src/main.py

```diff
@@ -98,18 +98,23 @@
     # such as unsafe-inline or unsafe-eval is highly discouraged, as these
     # may lead to XSS attacks.
     'csp_policy': {
-        # https://developers.google.com/fonts/docs/technical_considerations
-        'font-src':    '\'self\' themes.googleusercontent.com '
-                       '*.gstatic.com',
+        # Fallback.
+        'default-src': '\'self\'',
+        # Disallow Flash, etc.
+        'object-src': '\'none\'',
+        # Google Analytics.
+        'script-src': '\'self\' https://www.google-analytics.com/analytics.js',
```

Comment:
Is there a summary / list somewhere of which browsers support full URL granularity, so we know how often this might fail given current browser versions / market share?